core/shared/src/main/scala/zio/console/Console.scala

```diff
@@ -44,7 +44,7 @@ object Console extends Serializable {
       final def putStr(line: String): UIO[Unit] =
         putStr(SConsole.out)(line)
 
-      final def putStr(stream: PrintStream)(line: String): UIO[Unit] =
+      private[this] final def putStr(stream: PrintStream)(line: String): UIO[Unit] =
```

Comment:
It's public API and we want to putStr to streams other than Console sometimes. Could you, please, revert this and two other occurrences in this file?